test/mlir/conversion/onnx_to_tosa/Math/Gemm_to_matmul_allattr.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: onnx-mlir-opt --shape-inference --convert-onnx-to-tosa %s -split-input-file | FileCheck %s
```

Comment:
All `gemm_to_matmul` related tests can go int he one file, given the `-split-input-file` support.